frontend/packages/console-shared/src/components/dashboard/status-card/HealthItem.tsx

```diff
@@ -21,7 +21,7 @@ const HealthItem: React.FC<HealthItemProps> = React.memo(
     return (
       <div
         className={classNames('co-status-card__health-item', className)}
-        data-test={`${title}-health-item`}
+        data-test={`${title.toLowerCase().replace(/\s/g, '-')}-health-item`}
```

Comment:
This is only applicable to us. So we should make changes to title before passing here. Thank making this a generic logic.